web/pandas/about/team.md

```diff
@@ -52,7 +52,15 @@ The project governance is available in the [project governance page]({{ base_url
 ### {{ workgroup.name }}
 
 <ul>
-    <li><b>Contact:</b> <a href="mailto:{{ workgroup.contact }}">{{ workgroup.contact }}</a></li>
+    <li><b>Contact:</b>
+        <script TYPE="text/javascript">
+            var email='{{ workgroup.contact }}'
+            document.write(
+              '<a href="mailto:' + email + '">'
```

Comment:
```suggestion
              '<a href="mailto:asp.' + email + '">'
```
`asp` for `anti spam prefix`, and the html document will be sent as `document.write('<a href="asp.coc@pandas.pydata.org" ...`